src/Vlingo.Actors/ProxyGenerator.cs

```diff
@@ -442,6 +442,11 @@ private string GetSimpleTypeName(Type type)
             return type.FullName ?? type.Name;
         }
 
+        private string TypeFullNameToString(string typeFullName)
+        {
+            return typeFullName.Replace("+", ".");
```

Comment:
What's the name with `+` inside ?

Reply:
Full name contains + instead of the dot when you get `type.FullName`, for example: `public class MyInterface__Proxy : Vlingo.Actors.Tests.MyClass+IMyInterface` is returned and then generated to the source code.

Reply:
That's probably to distinguish namespace from nested classes/interfaces

Reply:
ok